### PR TITLE
update form istoriaviazen

### PR DESCRIPTION
**Pull request by MarioAP**
User can edit istoriaviazen

### CODE REVIEW

File: map/templates/map/update_istoriaviazen.html [hunk 1]
```diff
@@ -0,0 +1,78 @@
+{% extends 'map/base.html' %}
+{% load widget_tweaks %}
+
+{% block title %}Hatama istoria viazen {% endblock %}
+
+{% block content %}
+{% load i18n %}
+
+    <!-- jQuery -->
+    <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
```
Comment (PeteCoward) [inline map/templates/map/update_istoriaviazen.html:10]: not needed

File: map/views.py [hunk 2]
```diff
@@ -51,6 +51,7 @@ def get_success_url(self):
 
 
 class ViazenUpdateView(UpdateView):
+    template_name = 'map/update_istoriaviazen.html'
```
Comment (PeteCoward) [inline map/views.py:54]: use the `viajen_form`

File: map/views.py [hunk 3]
```diff
@@ -40,6 +41,11 @@ class HatamaViazenView(CreateView):
     def get_success_url(self):
         return reverse_lazy('photo_viazen', args = (self.object.id,))
 
+    def get_context_data(self, *args, **kwargs):
+        context = super().get_context_data(*args, **kwargs)
+        context['hatama_viazen'] = ("Hatama Istoria Viazen")
```
Comment (eraulo) [inline map/views.py:46]: @MarioAP please change to english and trans

File: map/views.py [hunk 4]
```diff
@@ -51,10 +57,16 @@ def get_success_url(self):
...
 
 class ViazenUpdateView(UpdateView):
+    template_name = 'map/viajen_form.html'
     model = Istoriaviazen
     fields = ['title', 'description', 'date', 'creator']
     success_url = reverse_lazy('home')
 
+    def get_context_data(self, *args, **kwargs):
+        context = super().get_context_data(*args, **kwargs)
+        context['update_viazen'] = ("Atualiza Istoria Viazen")
```
Comment (eraulo) [inline map/views.py:67]: @MarioAP please change to english and add trans

File: map/templates/map/update_istoriaviazen.html [hunk 5]
```diff
@@ -0,0 +1,78 @@
+{% extends 'map/base.html' %}
```
Comment (eraulo) [inline map/templates/map/update_istoriaviazen.html:1]: @MarioAP please remove this file `update_istoriaviazen.html`

**Pull request merged**